docs/mine/connectivity.md

```diff
@@ -14,6 +14,17 @@ Filecoin miners, like participants in all peer-to-peer protocols, require a stea
 
 The following steps are highly recommended for all miners who wish to successfully accept storage and retrieval deals.
 
+## Setting multiaddresses in your miner info
+
+You can set the addresses that your miner can be connected to with the command:
+ 
+ ```
+ lotus-miner set-addrs <multiaddr_1> <multiaddr_2> ... <multiaddr_n>
+```
+
+This updates the on-chain record of your `MinerInfo`, which is looked up
+when a client attempts to make a deal with you. You can provide any number of addresses.
+
```

Comment:
Wanna maybe give an example of a multiaddr that a miner might set? and also maybe discuss how they might find these addresses